### PR TITLE
Skip the extra hoops of escaping arguments

### PR DESCRIPTION
**Pull request by eric**
Skip the extra hoops of escaping arguments -- the system does this itself.

For more details: http://bitmonkey.net/post/344842339

### CODE REVIEW

File: bin/gifme [hunk 1]
```diff
@@ -82,29 +76,24 @@ if files.first[0..3] == 'http'
...
-files = files.collect{|a| escape a}
-
-cmd  = "convert"
-cmd += " -delay #{options[:delay]}"
-cmd += " -loop 0"
-cmd += " -resize #{options[:width]}"
-cmd += " -layers OptimizeTransparency"
-cmd += " #{files.join(' ')}"
+cmd  = [ "convert" ]
+cmd << "-delay" << options[:delay].to_s
```
Comment (holman) [inline bin/gifme:80]: This stuff gets really gross fast.

Comment (eric) [inline bin/gifme:80]: You can also do this trick:

```
cmd += %W(-delay #{options[:delay]})
```

and it's almost no more ugly...

**Comment by holman** [comment]
Initially I went to do this (since the manual escaping was such a hack, admittedly) and I ended up with some really weird situation where `system` needed to get multiple arguments and I had to do something else to the last one and I think I needed far beyond what Ruby allowed for. But I think things have since changed in the code where I can do it again. I might take another look at refactoring (since I'm really not a fan of needing to shift everything on).

**Comment by eric** [comment]
Strange. I would be really interested in what you ran into if you ever dig it up or run into it again...

**Comment by holman** [comment]
I think it was just trying to splat out the files array but still retain the first and last arguments to the method. Also maybe I was just retarded. That happens sometimes too.

**Comment by holman** [comment]
Alright, cool, I'll go with this. Just wish there were some more flexibility dropping down to `system` in Ruby; I always feel like I'm unhappy doing some of these things. Closed in b0eb91ad8a03a6e46165406a2690f9ec8fd8099e.

**Comment by eric** [comment]
Hmmmm... what sort of flexibility do you feel like you lose?

**Comment by holman** [comment]
I just wish Ruby was more shellish at times, when you get down to that level of code. And `system`, `exec`, and ``` all feel like bolted-on solutions at times.

**Pull request closed without merging**